.gitignore

```diff
@@ -1,16 +1,15 @@
 __pycache__
```

Comment:
I suggest we base our .gitignore on https://github.com/github/gitignore/blob/master/Python.gitignore This is nicely structured, has ignores for common IDEs (except PyCharm; we should also ignore .idea) and various possible directories for virtual environments.